crates/simulacrum/src/lib.rs

```diff
@@ -549,6 +549,29 @@ impl<T, V: store::SimulatorStore> ReadStore for Simulacrum<T, V> {
     }
 }
 
+impl<T: Send + Sync, V: store::SimulatorStore + Send + Sync> RestStateReader for Simulacrum<T, V> {
+    fn get_transaction_checkpoint(
+        &self,
+        _digest: &sui_types::digests::TransactionDigest,
+    ) -> sui_types::storage::error::Result<
+        Option<sui_types::messages_checkpoint::CheckpointSequenceNumber>,
+    > {
+        todo!()
```

Comment:
a reminder about this todo, in case it can be implemented now.

Reply:
This is in test-only code.